### PR TITLE
Merge sssd

### PR DESCRIPTION
**Pull request by mjg59**

### CODE REVIEW

**Comment by mjg59** [comment]
Depends on https://github.com/coreos/portage-stable/pull/412

**Comment by crawford** [comment]
Can you clean up those commit messages? (e.g. "sys-libs/ldb: sync from upstream")

**Comment by mjg59** [comment]
@crawford Done

**Comment by mjg59** [comment]
retest this please

File: coreos-devel/sdk-depends/sdk-depends-0.0.1.ebuild [hunk 1]
```diff
@@ -24,11 +24,14 @@ DEPEND="
 	coreos-base/hard-host-depends
 	coreos-base/coreos-sb-keys
 	coreos-devel/mantle
+	dev-libs/gobject-introspection
+        dev-libs/nss
```
Comment (mischief) [inline coreos-devel/sdk-depends/sdk-depends-0.0.1.ebuild]: change spaces to tabs

Comment (mjg59) [inline coreos-devel/sdk-depends/sdk-depends-0.0.1.ebuild]: Done

File: profiles/coreos/base/package.accept_keywords [hunk 2]
```diff
@@ -94,3 +94,5 @@ dev-util/checkbashisms
 # https://curl.haxx.se/docs/security.html
 =net-misc/curl-7.47.1 ~amd64 ~arm64
 
+# Older versions of sssd fail to build
+=sys-auth/sssd-1.13.1 ~amd64 ~ar64
```
Comment (mischief) [inline profiles/coreos/base/package.accept_keywords]: ~arm64

Comment (mjg59) [inline profiles/coreos/base/package.accept_keywords]: Fixed

**Comment by marineam** [comment]
LGTM

**Pull request merged**